File: langgraph/data/equiv/88.py
def arrange_sequence(sequence):
    if not sequence:
        return []
    
    reverse_order = (sequence[0] + sequence[-1]) % 2 == 0
    return sorted(sequence, reverse=reverse_order)

# method name: sort_array -> arrange_sequence
# parameter name: array -> sequence
# replaced the ternary operator with an if statement for clarity
# used a separate variable reverse_order to store the boolean value

File: langgraph/result/qwen/32b-eq2js/88.js
function arrangeSequence(sequence) {
    if (!sequence.length) {
        return [];
    }
    
    const reverseOrder = (sequence[0] + sequence[sequence.length - 1]) % 2 === 0;
    return sequence.sort((a, b) => reverseOrder ? b - a : a - b);
}